Sources/Core/Microsoft.StreamProcessing/Operators/Stitch/StitchPipe.cs

```diff
@@ -218,6 +218,7 @@ public override unsafe void OnNext(StreamMessage<TKey, TPayload> input)
                             this.batch[this.outputCount] = default;
                             this.batch.key.col[this.outputCount] = default;
                             this.batch.hash.col[this.outputCount] = 0;
+                            this.batch.bitvector.col[this.outputCount >> 6] |= 1L << (this.outputCount & 0x3f);
```

Comment:
>this.batch.bitvector.col[this.outputCount >> 6] |= 1L << (this.outputCount & 0x3f); [](start = 28, length = 83)

Is this the fix to Stitch you mention in the description?

Reply:
Yes - payloads were being considered even if it was a punctuation.